client/tests/integration/components/packages/pas-form/show-test.js

```diff
@@ -0,0 +1,42 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | packages/pas-form/show', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('Displays all proposed development types', async function(assert) {
+    this.set('packageHasAllTypes', {
+      pasForm: {
+        dcpProposeddevelopmentsitenewconstruction: true,
+        dcpProposeddevelopmentsitedemolition: true,
+        dcpProposeddevelopmentsiteinfoalteration: true,
+        dcpProposeddevelopmentsiteinfoaddition: true,
+        dcpProposeddevelopmentsitechnageofuse: true,
+        dcpProposeddevelopmentsiteenlargement: true,
+        dcpProposeddevelopmentsiteinfoother: true,
+        dcpProposeddevelopmentsiteotherexplanation: 'Esta bien',
+      },
+    });
```

Comment:
Why should we duplicate the tests for both components?

Reply:
I was actually really conflicted where to put these tests...

Playing devil's advocate: Would repeating the tests make the components more modular, though? 

Say one day I removed the ProposedDevelopmentTypesList component, however there was still the requirement that the show.hbs page shows a list of proposed development types. Removing the component and its related tests won't mean we have a hole in our tests.

Reply:
my instinct is that the application test is more important than the rendering test.

Reply:
ur call just pick 1 :)

Reply:
:) okay 👍